action.yml

```diff
@@ -143,6 +143,13 @@ runs:
         rustup target add x86_64-unknown-none
       shell: bash
 
+    # We do this in case there is toolchain skew between repos
+    - name: Install older rust toolchain(s)
+      if: ${{ (runner.os == 'Linux') }}
+      run: |
+        rustup toolchain install 1.81.0
```

Comment:
Do we need to add the x86_64-unknown-none target for the older version as well? Or does it use the same target for both versions?